### ISSUE TITLE
Querying sequences via REST API

### ISSUE THREAD

**Issue by apaytuvi** [opencb/cellbase#286]
I try to get a sequence via `REST API`.

When, in `mongodb`, I make this search:

`db.genome_sequence.find({"start":1, "sequenceName":"1"})`

I get:

```
{ "_id" : ObjectId("57f514ebb8d22e7a4c109e43"), "sequenceName" : "1", "chunkId" : "1_0_2k", "start" : 1, "end" : 1999, "sequenceType" : "chromosome", "assembly" : "SL2.50", "sequence" : "GGCTTGTCCGACCCATTTGGAAAGGTCAAACGAGCCTCAAAGCGAGCATACCCCTCATTTCGACGATTTTCGTGTGCTATAGCACACCATTTTTTGGGTGATCCGGATTCCGACGTCAAAAATTCCAAATTTTTTCGTGGTCGTCCGTCAAGACCTTGTCTATGCATACGGTTGGCCATCACGGCTTGTCCGACCCATTTGGAAGGTCAAACGAGCCCCGAAGCGAGCATACCCCTCCATTCGACGATTTTCGTGTGCTATAGCACACCATTTTTTGGGTGATCCGGATTCCGACGTCAAAAATGCAAAAAGATTTCGTGGACGTCCGTCAATATATATATATATATATATATATATCTTATTATATAAAGTATATATATAAGTTGAAATTTATTTTTCATTCATATCAATTAAATTTCAATTATTCTTAATGAATTTTCTAATTACACAACTATCTCATAACCCCAAACTAAAAAATGTGATGTATGAGATAGTTTATCCTAGCACAAATACTTCTGATTAGTTTCACTGTTCAAACCATTTATATTATACGAAAATAAATTTATCGTTATTAGAAGGTAAGGGAATATTAACTTTAAACACCTATGAATATTAATATGTCTAAAGATTAGTGTCGGATCTGAATCAGCATTTGAATTATTGCTGGCATTTTCTATCTTTGTAAAAGGAACCTTAATTGCGTATATGAGTTGGGAAGATCTAGTTCTGGAAAGTGATTTCGTAATTAAGATCATAATTTGTTTAATTACTACTGTAACCTAAATTAGAAACCTTACTTTAAGAAGGATAACATTTTTTTTGAATCGGTTATGTTCTAACTACATGTATTTAGTTTATAAGGTAATATTTTTTTTGAAGATAAGACTTTCTTTTACGGAGCTTTAAAAGCCTGATGAAAAAGCAAAAAAAAATTATAATAGTTCAGTTGGTTTAGAGGGTGATCATTCACAAAAGTGATTTGAGATCAATCCCTTCTTAATGACTTCTAAATAGTTTTTTTGTGGTGGCTAAGTTGGTTTGGAGGGTGATCATTTACATAATTGATTTTTGATCGATCCCTCCTTAATGCCTTTTAAATAGTCTTTTGGGGTGGCTAAGTTGATTTGAAGGGTGATCATTCACGGGGATGATTAGGGATCAATCCCTCCTTAGTAGCTTTTAAACGGTCCTTTAGAATCGCTATATTGATTTGGAGGGTGGTCATTCACACGGATGACCCGATATCAATTCCTTCTCAATGCCTTATGAATTAAGCCTATTGCACAGAGCTTGCCTGGTGCAATTTACGTCCTCGATCAGGTTTGATAAAGTGGATGAAAAGTTCTGTATTTTCTAAAGTAATTAGTCATTTAAGATATAATAGATAAAAAGAATTGTTGTTTTTTAATTTGTTAAATTGAACAAGTAATTAAAAATAACTATAAAAGAAAAACCGAACAAATAAAATAGGACAGAATAAGTATTTGATTACACGGTTGAATGATTTTTTCATTGGTTTTTTAAATACAACTTTTACACATTAAAATGAAGTTATCGGAGATCACTACTATTTGAATTTTGATGTCCATCATGTGGTTCAAGATTTCTGTGCTGCAAAATTTATGGAAAAAAACATCCGTTACTATCTTTTCTTCTTTTCTTTGTAAGTATTTAGAAAAGGGAAAAGGATTAAATATACTCCTAAACTATTTGAAAAGGTTTAGGATATATCCTCCGTTTAAAGCTTGGTTTACTCATGCCCTCGCCGTCCAACTTTTGGTCCAGCTTTTCTCTTATAGGTGTAAGTTGTCATGTTGGATATATCCAACTCATTTTTCATTTCTTTAAATGTCAAATAGAATTGTCATGTCATTTTGACCTTACCACATGATATTTATATGAAAATGGAAAAATATTTGGTCTCATAAACACCTAATTCAACCAATAAATCAACCCTCTTTTAAATAAATTATCCGACCGATTTTTAACAATTTTGTTT", "_chunkIds" : [ "1_0_2k" ] }
```

However, when I try to access to `/cellbase/webservices/rest/v4/slycopersicum/genomic/region/1:1-1999/sequence?assembly=GCA_000188115.2`, It thows the following error:

```
{"apiVersion":"v4","warning":"","error":"java.lang.StringIndexOutOfBoundsException: String index out of range: 2000","queryOptions":{"exclude":["_id","_chunkIds"],"metadata":true,"include":null,"limit":1000,"skip":-1,"count":false,"assembly":"GCA_000188115.2"},"response":[{"id":"","time":0,"dbTime":-1,"numResults":-1,"numTotalResults":-1,"warningMsg":"Future errors will ONLY be shown in the QueryResponse body","errorMsg":"DEPRECATED: java.lang.StringIndexOutOfBoundsException: String index out of range: 2000","resultType":"","result":[]}]}
```

**Comment by javild** [comment]
Thanks @apaytuvi for reporting this. It's a bug.

Will be soon fixed in 4.0.1

**Comment by javild** [comment]
Fixed now @apaytuvi 
v4.0 was released some days ago. Please, download the new release from branch hotfix-4.0 and try again.
